ethstats/cron.go

```diff
@@ -0,0 +1,25 @@
+package ethstats
+
+import (
+	"time"
+)
+
+func (s *State) InitCleanCRON(days int) error {
+	go func() error {
+		every := time.Hour * 24
+		ticker := time.NewTicker(every)
```

Comment:
What happens if we stop the service? It starts to count again for 24 hours to pass?

Reply:
Yes it counts to 24 hours.